app/utils/import_export/read_csv.py

```diff
@@ -9,3 +9,13 @@ def read_table_names(file_path):
             table_names.append(line)
         csvFile.close()
     return table_names
+
+def read_column_names(file_path_column):
```

Comment:
We dont need this method. Above method can be reused after refactoring / renaming with a generic read method.

Reply:
yeah